dev-guide.md

```diff
@@ -0,0 +1,165 @@
+
+## Projects & Code Repositories
+
+There are a number of projects involved in the K8ssandra ecosystem.
+
+The majority of the projects developed and contributed to through K8ssandra can be found within the [k8ssandra GitHub Organization](https://github.com/k8ssandra).
+
+### k8ssandra/k8ssandra
+
+`k8ssandra/k8ssandra` represents the high-level umbrella project for the ecosystem.  This is primarily a packaging of Helm charts and testing capabilities.  This is the project from where K8ssandra is assembled and deployed.
+
+[GitHub Repository](https://github.com/k8ssandra/k8ssandra)
+
+### k8ssandra/k8ssandra-operator
+
+`k8ssandra/k8ssandra-operator` represents the next generation of the K8ssandra implementation.  This project implements a Kubernetes operator that is responsible for managing the full deployment of K8ssandra across multiple clusters.
+
+[GitHub Repository](https://github.com/k8ssandra/k8ssandra-operator)
+
+### k8ssandra/cass-operator
+
+`k8ssandra/cass-operator` is the Kubernetes operator responsible for managing the deployment of Apache Cassandra within a K8ssandra cluster.  This project was originally developed under `datastax/cass-operator` and migrated to the K8ssandra organization.
+
+[Github Repository](https://github.com/k8ssandra/cass-operator)
+
+[Docker Hub Repository](https://hub.docker.com/r/k8ssandra/cass-operator)
+
+### k8ssandra/management-api-for-apache-cassandra
+
+`k8ssandra/management-api-for-apache-cassandra` is a sidecar service layer that attempts to build a well supported set of operational actions on Apache Cassandra nodes that can be administered centrally.  This is the layer through which the Apache Cassandra cluster nodes are managed.
+
+[GitHub Repository](https://github.com/k8ssandra/management-api-for-apache-cassandra)
+
+[Docker Hub Repository](https://hub.docker.com/r/k8ssandra/cass-management-api)
+
+### k8ssandra/medusa-operator
+
+`k8ssandra/medusa-operator` is the Kubernetes operator responsible for managing backup and restore capabilities for Apache Cassandra using Medusa.
+
+[GitHub Repository](https://github.com/k8ssandra/medusa-operator)
+
+[Docker Hub Repository](https://hub.docker.com/r/k8ssandra/medusa-operator)
+
+### k8ssandra/reaper-operator
+
+`k8ssandra/reaper-operator` is the Kubernetes operator responsible for managing repair capabilities for Apache Cassandra using Reaper.
+
+[GitHub Repository](https://github.com/k8ssandra/reaper-operator)
+
+[Docker Hub Repository](https://hub.docker.com/r/k8ssandra/reaper-operator)
+
+### thelastpickle/medusa
+
+`thelastpickle/medusa` is the tool that K8ssandra has chosen to manage backup and restore capabilities for the Apache Cassandra within the stack.  Medusa itself has a vibrate project community that the K8ssandra team regularly contributes to - both to help improve Medusa in general and also to provide features within K8ssandra.
```

Comment:
Correction needed here.

Reply:
I took care of this one as well @jeffbanks thanks for the catch!